tests/utils/test_grid.py

```diff
@@ -0,0 +1,42 @@
+import numpy as np
+import pytest
+
+import reciprocalspaceship as rs
+
+
+@pytest.mark.parametrize("sample_rate", [0.5, 1.0, 2.5, 3.0, 5.7])
+@pytest.mark.parametrize("dmin", [10.0, 7.0, 5.0, 3.0, 2.0, None])
+def test_get_gridsize(mtz_by_spacegroup, sample_rate, dmin):
+    """
+    Test rs.utils.get_gridsize() with fmodel data
+    """
+    dataset = rs.read_mtz(mtz_by_spacegroup)
+
+    if dmin is None:
+        dmin = dataset.compute_dHKL().dHKL.min()
+
+    if sample_rate < 1.0:
+        with pytest.raises(ValueError):
+            rs.utils.get_gridsize(dataset.cell, dmin=dmin, sample_rate=sample_rate)
+
+    else:
+        result = rs.utils.get_gridsize(dataset.cell, dmin=dmin, sample_rate=sample_rate)
+
+        # shape of output
+        assert isinstance(result, list)
+        assert len(result) == 3
```

Comment:
as long as we're doing this, you may as well make sure the list contains ints. something like: 
```python
assert all([isinstance(i, int) for i in result])
```

Reply:
good idea -- just added this.